components/backup/production/kflux-ocp-p01/backup-s3-credentials-patch.yaml

```diff
@@ -0,0 +1,3 @@
+- op: replace
+  path: /spec/dataFrom/0/extract/key
+  value: integrations-output/terraform-resources/appsrep09ue1/stonesoup-infra-prod-backup/backup-kflux-ocp-p01
```

Comment:
@manish-jangra Did you create the s3 and kms in prod2 account?

Reply:
not yet, I will do it today.

Reply:
Merge Requested submitted.